### ISSUE TITLE
Hardware issues are also tracked here?

### ISSUE THREAD

**Issue by asilrohit7** [FracktalWorks/Julia3GTouchUI#1]
Question

**Comment by vjvarada** [comment]
No, this is only for software, and basically, follows scrum for fixing issues by developers. 

But you will be amazed to know things you can do here, Check this out: https://github.com/blog/1633-3d-file-diffs
